lms/static/scripts/frontend_apps/utils/api.js

```diff
@@ -4,7 +4,8 @@
  */
 export class ApiError extends Error {
   constructor(status, data) {
-    const message = data.error_message || data.message || 'API call failed';
+    // If message is omitted, pass undefined to base class, not null.
+    const message = data.message || undefined;
     super(message);
```

Comment:
passing null here causes `this.message` to be the string "null". undefined seems more semantic which produces a result of ""